CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Added
+- Added request name to results variable passed to report. [#390](https://github.com/scanapi/scanapi/pull/390)
```

Comment:
```suggestion
- Request name to report. [#390](https://github.com/scanapi/scanapi/pull/390)
```

Just a small suggestion, in order to have the changelog entry more focused on the final user.